yt/visualization/profile_plotter.py

```diff
@@ -1455,7 +1455,7 @@ def set_xlim(self, xmin=None, xmax=None):
 
         >>> import yt
         >>> ds = yt.load('IsolatedGalaxy/galaxy0030/galaxy0030')
-        >>> pp = yt.PhasePlot(ds.all_data(), 'density', 'temperature', 'cell_mass')
+        >>> pp = yt.PhasePlot(ds.all_data(), 'density', 'temperature', ('gas', 'mass'))
```

Comment:
Here and below can you make all three of these tuples?

Reply:
Sure, I can.  I was just trying to make this PR directly address a single issue.  But I can start slowly modifying things to be full-tuple field designation.

Reply:
No need to do it everywhere if you don't want to but since you're already touching these lines I think it makes sense to update them in this PR.